features/user/account/renewing_account_after_deletion.feature

```diff
@@ -0,0 +1,18 @@
+@account
+  Feature: Registering account again after it has been deleted
```

Comment:
We have a new tool on master now. To format features use `bin/kawaii gherkin:fix --align=left features/`